### ISSUE TITLE
document + test workflowsets compatibility more thoroughly

### ISSUE THREAD

**Issue by simonpcouch** [tidymodels/stacks#120]

**Comment by mattwarkentin** [comment]
FWIW, when I add the `workflow_set` object, there are no "model configurations". Seems to be the issue...but why no configurations?
```
# A data stack with 2 model definitions and 0 candidate members:
#   std_LASSO: 0 model configurations
#   std_RandFor: 0 model configurations
# Outcome: class (factor)
```

**Comment by simonpcouch** [comment]
Oh, wow—thanks for chiming in. This is a new issue to me.

The issue arises here:

https://github.com/tidymodels/stacks/blob/4843385f031790cb16c6ab5713926551f9897cde/R/add_candidates.R#L282-L285

The logic here selects _class probability_ predictions and the true outcome and gets rid of columns with `.pred_class` (i.e. the columns with _hard class_ predictions), as the ensemble doesn't make use of hard class predictions in stacking models. {tune} (or some other dependency) labels hard class prediction columns as `.pred_colname`, which is where your issue comes from—your outcome column is named `class`.

So, quick fix—rename that `class` column. I'll open an issue in a moment to handle columns named `class` more elegantly in the future. :)

**Comment by mattwarkentin** [comment]
Oh, wow. So this is just an annoying edge case because I was playing around with data from `sim_classification()`?

**Comment by simonpcouch** [comment]
Yup! But any outcome named `class` will trigger it.😅 Woopsies.

**Comment by simonpcouch** [comment]
Addressed in #161, #164, and #165. :)
